application/views/header.php

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en" class="h-100">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="<?php echo base_url('assets/css/bootstrap.min.css'); ?>">
+    <link rel="stylesheet" href="<?php echo base_url('assets/css/style.css'); ?>">
+
+
+    <title><?php echo $page_title; ?></title>
+   
+  </head>
+  <body class="d-flex flex-column h-100">
+    
+    <div class="container pt-4 pb-4">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light rounded">
+            <a class="navbar-brand" href="#">CI First App</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample09" aria-controls="navbarsExample09" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+    
+            <div class="collapse navbar-collapse" id="navbarsExample09">
+                <ul class="navbar-nav mr-auto">
+					<li class="nav-item active">
+						<a class="nav-link" href="<?php echo base_url(); ?>">Home</a>
+					</li>
+					<li class="nav-item active">
+						<a class="nav-link" href="<?php echo site_url('users/create'); ?>">Create An User</a>
+					</li>
```

Comment:
Please fix indentation here and other place also.

Reply:
Yes, sir